freqtrade/freqai/prediction_models/ReinforcementLearner_multiproc.py

```diff
@@ -34,17 +35,23 @@ def set_train_and_eval_environments(self, data_dictionary: Dict[str, Any],
         train_df = data_dictionary["train_features"]
         test_df = data_dictionary["test_features"]
 
+        env_info = {"live": False}
+        if self.data_provider:
+            env_info["live"] = self.data_provider.runmode in (RunMode.DRY_RUN, RunMode.LIVE)
```

Comment:
I think we have `self.live` already available here since `ReinforcmentLearner*` is a child of `IFreqaiModel` see [here](https://github.com/freqtrade/freqtrade/blob/fix/multiproc-dp/freqtrade/freqai/freqai_interface.py#L133).

```python
env_info["live"] = self.live
```

Reply:
That's better, thanks.